src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
```

Comment:
I see that `posts` from the props can be empty array, maybe handle this in template and show proper info for user.
Maybe something like "Sorry, Your posts list is empty."